recipes/kfp-notebook/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "kfp-notebook" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/49/72/675e1176800c346939e396f80452ea8748cb0b0c61cdac4552ae0a0126c4/kfp_notebook-{{ version }}-py3-none-any.whl
+  sha256: 02d207837a26530345e78b54c910c8d51a92fa005cf9910761e14a9672f0677c
```

can you please use the sdist package here instead?
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 72d57cb3a32e3740f50fb0a5d5c36703a855510fd14b12b01c2273ab9c8b1d6d
```